flows/finance-data/.prefectignore

```diff
@@ -0,0 +1,41 @@
+# prefect artifacts
```

Comment:
> By default, Prefect uploads all files in the current folder to the configured storage location (local by default) when you build a deployment.
> 
> If you want to omit certain files or directories from your deployments, add a .prefectignore file to the root directory. .prefectignore enables users to omit certain files or directories from their deployments.
> 
> Similar to other .ignore files, the syntax supports pattern matching, so an entry of *.pyc will ensure all .pyc files are ignored by the deployment call when uploading to remote storage.

Interesting. I wonder if this can live in the root of the repo and then we don't need to add one to every directory? Also, the prefect docs are not clear, but since we're using github storage i don't think the deployment is actually uploading files anywhere, right?

Reply:
I'm leaning towards .gitignoring the .prefectignore. I'll put it on the agenda for Tuesday

Reply:
@johnclary that's correct, maybe that's something can configure but I doubt we want Prefect auto-committing stuff to github. Currently, I get a warning saying it's not configured to upload to our storage location. I'm fine with gitignoring it!